lib/src/isolate_manager/isolate_manager_web/isolate_manager.dart

```diff
@@ -0,0 +1,50 @@
+import 'dart:async';
+
+import 'package:dash_kit_network/dash_kit_network.dart';
+import 'package:dash_kit_network/src/isolate_manager/isolate_manager_interface.dart';
+
+///  The IsolateManager class implements the IsolateManagerInterface.
+///
+/// Provides an interface for managing isolates in the DashKit network library
+/// in web environment.
+class IsolateManager extends IsolateManagerInterface {
+  /// The start() method sets the isolate status to initialized.
+  ///
+  /// If `start()` is called multiple times, an assertion error is thrown.
+  /// In web environment the `start()` method doesn't do anything.
+  @override
+  Future<void> start() async {
+    await super.start();
+    handleSendPortMessage(null);
+
+    return;
+  }
+
+  /// The stop() method sets the isolate status to stopped.
+  @override
+  void stop() {
+    super.stop();
+
+    return;
+  }
+
+  /// Implementation for web environment where isolates aren't available.
+  ///
+  /// Instead the `send` method executes the mapper function and returns result.
+  @override
+  Future<FutureOr<T>> send<T>({
+    required Response<dynamic> response,
+    required FutureOr<T> Function(Response<dynamic>) mapper,
+  }) async {
+    await super.send(response: response, mapper: mapper);
+
+    return await mapper(response);
```

Comment:
```suggestion
    return mapper(response);
```

do we really need to create one more microtask here?

Reply:
Was changed.